src/Sql/dbo/Vault/Stored Procedures/Cipher/CipherDetails_Update.sql

```diff
@@ -18,7 +18,8 @@
     @DeletedDate DATETIME2(2),
     @Reprompt TINYINT,
     @Key VARCHAR(MAX) = NULL,
-    @ArchivedDate DATETIME2(7) = NULL
+    @ArchivedDate DATETIME2(7) = NULL,
+    @Archives NVARCHAR(MAX) = NULL -- not used
```

Comment:
🎨 Similarly, we can probably just leave off the `@Archives` argument entirely to avoid adding to the confusion of all these `-- not used` properties